src/components/HOCs/WithSearch/WithSearch.test.js

```diff
@@ -225,7 +225,8 @@ test("moving the map does perform new search if filtering within map bounds", ()
   const newState = _cloneDeep(basicState)
   newState.currentSearch.challenges.mapBounds = {bounds: [0,0,0,1]}
 
-  const didNewSearch = wrapper.instance().componentDidUpdate(newState)
+  // const didNewSearch = wrapper.instance().componentDidUpdate(newState)
+  wrapper.instance().componentDidUpdate(newState)
 
   //expect(didNewSearch).toBe(true)
```

Comment:
Did you try uncommenting this line instead to see if the test passes? If so I think we should do that. Currently this test doesn't assert anything.